components/Common/Notification/index.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { CodeBracketIcon } from '@heroicons/react/24/solid';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import { FormattedMessage } from 'react-intl';
+
+import Notification from './index';
+
+type Story = StoryObj<typeof Notification>;
+type Meta = MetaObj<typeof Notification>;
+
+export const Default: Story = {
+  args: {
+    duration: 500000,
```

Comment:
Is there a reason why the duration in storybook is not set to 5 seconds?

Reply:
the component will disappear after 5 seconds, so I thought of keeping a long duration for demonstration purposes.

Reply:
updated duration and added `open` prop

Reply:
Thanks, I personally prefer a shorter duration so that it's easier to verify that it does automatically close and pauses closing on hover & focus.